### ISSUE TITLE
InitialData does not work with TextArea (Input multiline)

### ISSUE THREAD

**Issue by rvieceli** [Rocketseat/unform#129]
**Description of bug**
When passing inicialData object, all inputs are filled, except textinput

**To Reproduce**
```
const inicialData = {
  id: 2,
  title: 'New',
  description: 'Description of new Meetup',
  location: 'Here',
  date: '2019-07-10T13:00:00.000Z',
}

<Form initialData={meetup}>
  <Input type="text" name="title" placeholder="Título" />
  <Input multiline name="description" placeholder="Descrição" />
  <Input type="date" name="date" placeholder="Data" />
  <Input type="text" name="location" placeholder="Localização" />
</Form>
```

The result: textarea does not have the value
![image](https://user-images.githubusercontent.com/2790845/63790195-31c09180-c8cf-11e9-8ec1-6b4f03718f05.png)

**Expected behavior**
Textarea with value of the inicialData
![image](https://user-images.githubusercontent.com/2790845/63790432-9e3b9080-c8cf-11e9-9cc0-11f4b6a3aa5f.png)

**Exception or Error**
Does not have

**Environment:**

 - OS: Windows 10
 - Browser: Firefox (68.0.2)

**Comment by avitorio** [comment]
I'm getting the same error on OSX, Chrome.

**Comment by Artenes** [comment]
Same thing here on Windows 7 and Chrome Versão 77.0.3865.90 64 bits

**Comment by diego3g** [comment]
Unform released version 2.0 🎉

Now it does not include visual components inside the library, so you cannot import Input, Select or any other element from unform package.

Instead of that, you can now easily create your own custom input sources using the `useField` API.

Check [this guide](https://unform.dev/recipes/simple-input) for an easy example.

I recommend you creating an input for textearea apart from the simple input, so you can have more control of the styles between both of them.
